src/ape/pytest/plugin.py

```diff
@@ -14,33 +14,42 @@
 
 
 def pytest_addoption(parser):
-    parser.addoption(
-        "--showinternal",
-        action="store_true",
-    )
-    parser.addoption(
+    def add_option(*names, **kwargs):
+        try:
+            parser.addoption(*names, **kwargs)
+        except ValueError as err:
+            if "already added" in str(err):
+                # A different plugin added an option
+                # with this name. It should be ok to share it.
+                logger.warning(f"Multiple pytest options named '{names[0]}'.")
```

Comment:
ugh, no longer liking this because now we are potentially telling other plugins wrong things...
perhaps it is better to raise a nicer error instead.